backend/bin/compile_requirements.sh

```diff
@@ -4,10 +4,14 @@ pip-compile --no-emit-index-url --upgrade multi-account/requirements.in
 pip-compile --no-emit-index-url --upgrade multi-account/requirements-dev.in
 pip-compile --no-emit-index-url --upgrade compact-connect/requirements.in
 pip-compile --no-emit-index-url --upgrade compact-connect/requirements-dev.in
+pip-compile --no-emit-index-url --upgrade compact-connect/lambdas/custom-resources/requirements.in
+pip-compile --no-emit-index-url --upgrade compact-connect/lambdas/custom-resources/requirements-dev.in
 pip-compile --no-emit-index-url --upgrade compact-connect/lambdas/license-data/requirements.in
 pip-compile --no-emit-index-url --upgrade compact-connect/lambdas/license-data/requirements-dev.in
 pip-compile --no-emit-index-url --upgrade compact-connect/lambdas/provider-data-v1/requirements.in
 pip-compile --no-emit-index-url --upgrade compact-connect/lambdas/provider-data-v1/requirements-dev.in
```

Comment:
If I go to a provider route like `https://app.test.compactconnect.org/aslp/LicenseeDashboard`, it redirects me to the staff login page, rather than the provider login page.

Reply:
Good catch, this is a known issue with how we implemented the multi user pool login the first time around.

I have created a ticket here that will fix this and all of the other edge cases issues introduced by the initial implementation: https://github.com/csg-org/CompactConnect/issues/321